generator/src/main/java/com/algorand/sdkutils/generators/OpenApiParser.java

```diff
@@ -135,6 +135,7 @@ TypeDef getType(
             case "byte":
             case "base64":
             case "digest":
+            case "TEALProgram":
```

Comment:
it is weird, why do we need it? it is not in the spec

Reply:
It is. In indexer spec file.

Reply:
I still do not understand why do we need it at all.

Reply:
In the spec file, two format specifications are given: x-algorand-format and format:
        "approval-program": {
          "description": "\\[approv\\] approval program.",
          "type": "string",
          "format": "byte",
          "x-algorand-format": "TEALProgram"

At this point, the generator first looks at format, i.e., picks up "byte". Then, if x-algorand-format is specified, it overrides format, i.e.picks up "TEALProgram". Changing this behavior requires changes to other properties in the spec file.

Then, looks for a rule for TEALProgram. If one does not exist, then picks up type, which is "string".
This change adds a rule for TEALProgram, so that it does not go and pick "string". 

In my view, having TEALProgram is a good practice, since we are being more specific. 
What needs to be cleaned up is the use of specifications such as "byte", ore "base64", which are more ambiguous.